helm/private/helm_import_authn.bzl

```diff
@@ -11,8 +14,151 @@ _WWW_AUTH = {
     },
 }
 
+def _file_exists(repository_ctx, path):
+    result = repository_ctx.execute(["stat", path])
+    return result.return_code == 0
```

Comment:
Isn't there an "exists" method in path objects? I'd like to avoid assuming "stat" exists.